src/Directory.Build.props

```diff
@@ -71,6 +71,25 @@
     <DefineConstants>$(DefineConstants);ENCLAVE_SIMULATOR</DefineConstants>
   </PropertyGroup>
 
+  <!-- Packaging -->
+  <PropertyGroup>
+    <DebugType>portable</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUnTrackedSources>true</EmbedUnTrackedSources>
+  </PropertyGroup>
+  <!-- Packaging Extension CI Builds -->
+  <PropertyGroup Condition="'$(TF_BUILD)' == 'true'">
```

Comment:
Can you update the [BUILDGUIDE.md](https://github.com/dotnet/SqlClient/blob/main/BUILDGUIDE.md) and explain how to use it with example(s)?

Reply:
This block with TF_BUILD was deleted as the NugetPackProperties in build.proj is the actual properties used.
TF_BUILD condition is used in AzureDevOps for Deterministic build.

Reply:
If it's used my original query still should be addressed.

Reply:
The TF_BUILD condition and its enclosed property definitions no longer exists.

The  NugetPackProperties definition has the actual properties used to create the nupkg file.

Reply:
BUILDGUIDE.md was updated to explain what TF_BUILD is with example usage.